app/src/main/java/com/strv/movies/ui/moviedetail/MovieDetailScreen.kt

```diff
@@ -82,29 +87,46 @@ fun MovieTrailerPlayer(videoId: String, progressSeconds: MutableState<Float>) {
 
 @Composable
 fun MoviePoster(movie: MovieDetail) {
-    AsyncImage(
-        model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
-        contentDescription = stringResource(id = R.string.movie_image),
-        modifier = Modifier
-            .padding(top = 16.dp)
-            .size(120.dp)
-    )
+    Column {
+
+        AsyncImage(
+            model = "https://image.tmdb.org/t/p/w500${movie.posterPath}",
+            contentDescription = stringResource(id = R.string.movie_image),
+            modifier = Modifier
+                .padding(top = 16.dp)
+                .size(120.dp),
+        )
+        Text(
+            movie.title,
+            style = MaterialTheme.typography.subtitle2,
+            maxLines = 1,
+            overflow = TextOverflow.Ellipsis, // It will add "..." to the end
+            textAlign = TextAlign.Center,
+            color = Color.Red
+        )
+    }
 }
 
 @Composable
-fun MovieInfo(movie: MovieDetail) {
-    Column {
+fun MovieInfo(movie: MovieDetail, weight: Modifier) {
+    Column(
+        modifier = Modifier
```

Comment:
You need to apply the modifier with the weight to the `Column` (because its a container for all the content in `MovieInfo`). Then the weight will finally work and you can remove `Modifier.width(240.dp)' for down there completely. 😉  
```suggestion
fun MovieInfo(movie: MovieDetail, modifier: Modifier) {
    Column(
        modifier = modifier
```

Reply:
oh yes yes, sorry sorry , done